frontend/src/hooks/pims-api/useApiHealth.test.ts

```diff
@@ -0,0 +1,62 @@
+import { renderHook } from '@testing-library/react-hooks';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { useApiHealth } from '.';
+
+const mockAxios = new MockAdapter(axios);
+
+describe('useApiHealth testing suite', () => {
+  beforeEach(() => {
+    mockAxios.onGet('health/env').reply(200, defaultVersion);
+    mockAxios.onGet('health/live').reply(200, defaultLive);
+    mockAxios.onGet('health/ready').reply(200, defaultReady);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('Has endpoints', () => {
```

Comment:
I'm not a big fan of tests like this as you are essentially testing the structure of the code and not the functionality, but personal preference maybe.